examples/example01_vdx.py

```diff
@@ -0,0 +1,38 @@
+import pytrip as pt
+from pytrip.utils import cubeslice
+
+# first define some paths and other important parameters
+patient_name = "TST000000"
+ctx_path = "/home/bassler/Projects/CTdata/TST000/TST000000.ctx"
+vdx_path = "/home/bassler/Projects/CTdata/TST000/TST000000.vdx"
+my_target_voi = "GTV"
+
+# load CT cube
+my_ctx = pt.CtxCube()
+my_ctx.read(ctx_path)
+
+# load VOIs
+my_vdx = pt.VdxCube("", my_ctx)  # my_vdx is the object which will hold all volumes of interest and the meta information
+my_vdx.read(vdx_path)  # load the .vdx file
+print(my_vdx.get_voi_names())  # show us all VOIs found in the .vdx file
+
+# next pick a the proper VOI which we want to plan on
+target_voi = my_vdx.get_voi_by_name(my_target_voi)  # Select the requested VOI from the VdxCube object
+
+# dos cube representing VOI, it is filled with zeros outside the volume and with 1000 inside
+voi_cube = target_voi.get_voi_cube()
+
+# mask object, same shape as dos and voi cube, true inside volume, false outside
+mask = (voi_cube.cube == 1000)
+
+# make a masked CT - set Hounsfield units to zeros inside the volume
+my_ctx.cube[mask] = 0
```

Comment:
"The mask object and the CTX cube have similar dimensions (they are infact inherited from the same top level class). Therefore we can apply the mask cube to the ctx cube and work with the values. For instance we can set all HUs to zero within the Voi:"
# my_ctx.cube[mask] = 0
# or set it to some special based on an algorithm
# my_ctx_cube[mask] += 10  # increase HUs by 10
my_ctx.cube[mask] = <we need some index dependent operation here> # add random noise to the Voi